hack/test.bash

```diff
@@ -15,3 +15,18 @@ IMPORT_DURING_SOLVE=${IMPORT_DURING_SOLVE:-false}
 go test -race \
     -ldflags '-X github.com/golang/dep/cmd/dep.flagImportDuringSolve=${IMPORT_DURING_SOLVE}' \
     ./...
+
+if ! ./dep status -out .dep.status.file.output; then exit 1; fi
+if ! ./dep status > .dep.status.stdout.output; then
+   rm -f .dep.status.file.output
+   exit 1
+fi
+if ! diff .dep.status.file.output .dep.status.stdout.output; then
+  diffResult=1
+else
+  diffResult=0
```

Comment:
Hi @stanguturi,

There's no real reason to set the `diffResult` to 0, is there?

Reply:
Oh wait. Yes there is. Duh!